apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -498,39 +511,87 @@ public Flow(long maxByteCount, long maxElementCount, Action<Task> registerTaskFn
             /// has completed.
             /// </summary>
             /// <param name="byteCount">The number of bytes in the element associated with <paramref name="fn"/>.</param>
+            /// <param name="orderingKey">The ordering key for this message. Empty string if no ordering key.</param>
             /// <param name="fn">The function to execute.</param>
             /// <returns>A Task that completes once <paramref name="fn"/> has been scheduled for execution.</returns>
-            public async Task Process(long byteCount, Func<Task> fn)
+            public async Task Process(long byteCount, string orderingKey, Func<Task> fn)
             {
                 while (true)
                 {
                     lock (_lock)
                     {
                         if (IsFlowOk())
                         {
-                            // Flow is OK, so add for this element, and execute.
+                            // Flow is OK, so stop waiting.
+                            // Add to stats for this element.
                             _byteCount += byteCount;
                             _elementCount += 1;
+                            if (orderingKey.Length > 0)
```

Comment:
It's odd that there's no mention of what happens when there isn't an ordering key. Extra comment?

Reply:
Clarified.